app/routes/sign-up/index.tsx

```diff
@@ -0,0 +1,210 @@
+import { Form, Link, useActionData, useSearchParams } from '@remix-run/react';
+import { DataFunctionArgs, json, redirect } from '@remix-run/server-runtime';
+import { registerCustomerAccount } from '~/providers/account/account';
+import { XCircleIcon } from '@heroicons/react/solid';
+import {
+    extractRegistrationFormValues,
+    RegisterValidationErrors,
+    validateRegistrationForm,
+} from '~/utils/registration-helper';
+
+const DEMO_URL = 'https://readonlydemo.vendure.io/shop-api';
```

Comment:
This is duplicated here and in graphqlWrapper.ts. Can you extract it into the constants.ts file so it is defined only once?

Reply:
moved to constants.ts